tests/utils.test.ts

```diff
@@ -295,3 +296,13 @@ describe('UUIDT', () => {
         expect(uuidtString.slice(14, 18)).toEqual('0000')
     })
 })
+
+describe('sanitizeSqlIdentifier', () => {
+    it('removes quotes inside and adds quotes around identifier', () => {
+        const rawIdentifier = 'some_field"; DROP TABLE actually_an_injection;'
+
+        const sanitizedIdentifier = sanitizeSqlIdentifier(rawIdentifier)
+
+        expect(sanitizedIdentifier).toStrictEqual('"some_field; DROP TABLE actually_an_injection;"')
+    })
+})
```

Comment:
What about `firstKey = "haha; -- \\"` It would escape the closing quote. Combined with `otherKey = "; here we're outside the quotes... -- "` you could have some fun. Since we're the ones providing the keys, let's whitelist characters and remove everything not in some sane set.

Reply:
Yup, basically just made it remove all chars besides letters, digits and underscores, nuclear option